src/Emulator/Peripherals/Peripherals/UART/STM32F7_USART.cs

```diff
@@ -135,13 +135,13 @@ private void DefineRegisters()
                 .WithTag("ADD", 24, 8);
 
             Register.ControlRegister3.Define(this)
-                .WithTaggedFlag("EIE", 0)
+                .WithFlag(0, name: "EIE")
```

Comment:
same as above

Reply:
ok, so now I understand what is expected when defining register field as a flag, I will try to fix it according to your design